src/stylesheets/app.scss

```diff
@@ -0,0 +1,276 @@
+@import "base/typography";
+@import "base/vars";
+
+body{
+  margin: 0;
+  background-color: #d2e7e2; // only temporary
+}
+ul{
+  padding: 0;
+}
+sup{
+  color: #e95e67;
+  text-decoration: underline;
+}
+
+/*** Top Fold styles ***/
+#top-fold{
```

Comment:
I usually try to only style with classes and reserve id's for javascript. ID's can be too specific and hard to override later, especially when you're nesting them like you have below https://css-tricks.com/specifics-on-css-specificity/